services/tracking/tracker.py

```
@@ -82,162 +87,243 @@ def __init__(
-            tid  = int(t.track_id)
-            # ── ReID matching ─────────────────────────────────────
-            # ── ReID matching ─────────────────────────────────────
-    if hasattr(t, "features") and t.features:
+        if det.label != "person":
+            continue
 
-     new_embedding = t.features[-1]
+        b = det.bbox
 
-    for lost_id, data in list(self._lost_embeddings.items()):
+        l, t = b.x1, b.y1
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)